### ISSUE TITLE
on-click/tap still works if disabled is true. 

### ISSUE THREAD

**Issue by matthiasgoossens** [fxedel/toggle-icon#21]
The on-tap event still works if you have disabled attribute set to true.
Don't know if this is expected behavior?

**Comment by fxedel** [comment]
Correct. For example, the `paper-icon-button` component sets `pointer-events: none` when disabled to avoid the click event being fired. I'll add this after migrating to Polymer 3.0 (#20) – shouldn't be that complicated.
